main/src/main/java/org/mobilitydata/gtfsvalidator/notice/RouteColorContrastNotice.java

```diff
@@ -19,6 +19,12 @@
 import com.google.common.collect.ImmutableMap;
 import org.mobilitydata.gtfsvalidator.type.GtfsColor;
 
+/**
+ * "The color difference between route_color and route_text_color should provide sufficient contrast
+ * when viewed on a black and white screen." (http://gtfs.org/best-practices/#shapestxt)
```

Comment:
The URL here is wrong - it should be http://gtfs.org/reference/static/#routestxt

Reply:
https://gtfs.mobilitydata.org/spec/gtfs-schedule#h.elenes7l8gmy